src/main/scala/uk/gov/homeoffice/drt/repository/ArrivalRepository.scala

```diff
@@ -27,7 +32,7 @@ trait ArrivalRepositoryI[F[_]] {
 
   def sessionPool: Resource[F, Session[F]]
 
-  def findArrivalsForADate(queryDate: LocalDateTime): F[List[ArrivalTableData]]
+  def findArrivalsForADate(origins: List[String], queryDate: LocalDateTime): F[List[ArrivalTableData]]
```

Comment:
Should this be called `findArrivalsForOriginsAndDate` as it takes both parameters?